main.py

```diff
@@ -83,4 +85,13 @@ def run_menu():
 
 if __name__ == '__main__':
     # run_menu()
-    gpu.gpu_power(221, 40, 200)
+    val = 100003157
+    start = time.time()
+    x = (miller_rabin_cpu(val, 100000))
+    end = time.time()
+    print(x, end-start)
+    start = time.time()
+    y = gpu.gpu_power(val, 100000)
+    end = time.time()
+    print(y, end-start)

```

Comment:
po mergu ta część powinna pozostać funkcjonalna - czyli to powinno być menu. Proponuje inny plik, testowy najlepiej. W sumie to może być na inny, następny task, a to bym to stąd usunął. Akurat będzie coś do puszczenia przed prezką w piatek